examples/pinning/readme.md

```diff
@@ -22,7 +22,7 @@ ipfs pin ls --type=all
 As you may have noticed, the first `ipfs pin rm` command didnt work, it should
 have warned you that the given hash was "pinned recursively". There are three
 types of pins in the ipfs world; direct pins, which pin just a single block, and
-no others in relation to it. recursive pins, which pin a given block and all of
+no others in relation to it. Recursive pins, which pin a given block and all of
```

Comment:
This isn't the start of a sentence. It's the second of the three items in a list: direct pins, recursive pins, and indirect pins. Instead, the `;` should be `:` and the `.` should be `;` (to separate items that themselves contain commas).